pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -134,6 +134,10 @@ var invalidPluginsByVersion = []invalidPlugins{
 		schemeGroupVersion: v1beta3.SchemeGroupVersion.String(),
 		plugins:            []string{},
 	},
+	{
+		schemeGroupVersion: v1.SchemeGroupVersion.String(),
+		plugins:            []string{"SelectorSpread"},
```

Comment:
is there a test making sure you can't use a SelectorSpread plugin with a v1 config?

Reply:
yes, it's in pkg/scheduler/apis/config/validation/validation_test.go